extensions/positron-python/src/client/terminals/codeExecution/helper.ts

```diff
@@ -92,12 +98,10 @@ export class CodeExecutionHelper implements ICodeExecutionHelper {
             const startLineVal = activeEditor?.selection?.start.line ?? 0;
             const endLineVal = activeEditor?.selection?.end.line ?? 0;
             const emptyHighlightVal = activeEditor?.selection?.isEmpty ?? true;
-            let smartSendSettingsEnabledVal = true;
-            const configuration = this.serviceContainer.get<IConfigurationService>(IConfigurationService);
-            if (configuration) {
-                const pythonSettings = configuration.getSettings(this.activeResourceService.getActiveResource());
-                smartSendSettingsEnabledVal = pythonSettings.REPL.enableREPLSmartSend;
-            }
+            // --- Start Positron ---
+            // This setting is hidden in favor of the Positron console.
+            const smartSendSettingsEnabledVal = true;
+            // --- End Positron ---
```

Comment:
This one was slightly more annoying since it isn't wrapped in a nice helper function! 🙂 

In general should I leave upstream code commented-out or just deleted?

Reply:
We don't have a hard and fast rule on this, but in general it's better to comment it out because it makes it more straightforward to see the before/after when attempting to reapply a change during a merge.